evm/utils.js

```diff
@@ -466,6 +470,21 @@ const getEVMAddresses = async (config, chain, keyID = '') => {
     return { addresses, weights, threshold };
 };
 
+const getAmplifierKeyAddresses = async (config, chainName, keyID = '') => {
+    const client = await CosmWasmClient.connect(config.axelar.rpc);
+    const key = await client.queryContractSmart(config.axelar.contracts.Multisig.address, {
+        get_key: { key_id: { owner: config.axelar.chains[chainName].contracts.MultisigProver.address, subkey: keyID } },
```

Comment:
```suggestion
        get_key: { key_id: { owner: config.axelar.contracts.MultisigProver[chains].address, subkey: keyID } },
```
not all axelar contracts will be per chain